Sources/TuistGraph/Graph/Graph.swift

```diff
@@ -27,14 +48,18 @@ public struct Graph: Equatable, Codable {
     /// A dictionary that contains the one-to-many dependencies that represent the graph.
     public var dependencies: [GraphDependency: Set<GraphDependency>]
 
+    /// A dictionary that contains the platform filters to apply to a dependency relationship
+    public var edges: [GraphEdge: PlatformFilters]
```

Comment:
Since the information about the `edges` is already contained in the `dependencies` dictionary, what about calling this one `edgesMetadata` or `dependenciesMetadata` to reflect that it contains metadata for edges?

Reply:
this could just be `dependencyPlatformFilters` since that's what it is representing, the platform filters for a given dependency relatioship.  I'm hesitant to use a broad name like `metadata` since this will only serve a specific purpose.

Reply:
Sounds good. We can iterate on the naming if we end up needing to include more information in the edge.